pages/about.html

```diff
@@ -0,0 +1,158 @@
+                src="../img/logo/kohai-high-resolution-logo-black-on-transparent-background.png"
+                alt="logo"
+                class="logo"
+            /></a>
+            <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+              <li class="nav-item">
+                <a class="nav-link fw-semibold text-dark" href="#location"
+                  >Location</a
+                >
+              </li>
+              <li class="nav-item">
+                <a class="nav-link fw-semibold text-dark" href="#archive"
+                  >Archive</a
+                >
+              </li>
+              <li class="nav-item">
+                <a class="nav-link fw-semibold text-dark" href="#sponsor"
+                  >Sponsors</a
+                >
+              </li>
+            </ul>
+            <button type="button" class="btn btn-outline-dark">Claim your badge</button>
+          </div>
+        </div>
+      </nav>
+      <div
+        class="headline-about-text d-flex flex-column justify-content-center align-items-center w-75 mx-auto my-5"
+      >
+        <q class="fs-5 fw-semibold"
+          >Hi noobs! You don't want to miss this one!</q
+        >
+        <h1 class="fw-bold text-uppercase">🦄Unicorn Con 2023🧚</h1>
+        <p
+          class="p-3 bg-white fw-bolder opacity-75 text-dark border border-dark p-2 border-opacity-10"
+        >
+          <span class="fw-bold fs-5">Kohai</span> inc. just surpassed itself
+          this time. An event without precedence will take place this year in
+          West Blue, Ohara of all places. Put on your favorite cosplay costume,
+          because we will bazinga like it is our last day.
+        </p>
+        <p class="fs-1 fw-bold text-dark">Mon 29 - Wen 31 May</p>
+        <p class="text-dark fw-semibold">@Colosseum Ohara West Blue</p>
+      </div>
+    </header>
+    <main>
+      <section
+        id="location"
+        class="d-flex flex-column align-items-center pb-3 bg-white border-bottom"
+      >
+        <h2 class="text-center fw-bolder pt-3 text-dark">Location</h2>
+        <span class="line-primary"></span>
+        <p class="text-center mt-3">
+          <span class="fw-bold fs-5">Unicorn Con 2023</span> will be held at
+          <br /><span class="fs-1">Colisseum Ohara, West Blue</span>
+        </p>
+        <button type="button"
+          class="btn btn-danger p-3 rounded-0 text-white fw-bold mx-auto text-uppercase"
+        >
+          Book your room
+        </button>
+      </section>
+      <section id="archive" class="d-flex flex-column align-items-center pb-3">
+        <h2 class="text-center fw-bolder pt-5 mb-3">Archives</h2>
+        <span class="line-primary"></span>
+        <ul class="mt-5 list-unstyled p-3 row text-light fw-bolder fs-1 ">
+          <li class="archive-item archive-one col-11 col-md-5 mx-auto my-1">Fairies Con 2018</li>
+          <li class="archive-item archive-two col-11 col-md-5 mx-auto my-1">Centaurs Con 2020</li>
```

Comment:
Kindly ensure that the text on the images are centered to match the design. :pray: